config/brands.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'brand_attribute_code' => env('BRAND_ATTRIBUTE_CODE', 'brand')
```

Comment:
```suggestion
    'brand_attribute_code' => 'brand',
```

The `env()` is for variables which could be different on environments. This settings wouldn't. Don't forget to change the readme.